RWKV-v4neo/train.py

```diff
@@ -122,8 +122,7 @@
     import numpy as np
     import torch
     from torch.utils.data import DataLoader
-    if "deepspeed" in args.strategy:
-        import deepspeed
```

Comment:
This modification is intentional so that deepspeed is loaded on-demand since it doesn't work on Windows.